utils/economic-checks/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+// This file is part of Gear.
+
+// Copyright (C) 202 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// #![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+use arbitrary::{Arbitrary, Error, Result, Unstructured};
+
+pub use targets::*;
+
+pub mod util;
+
+pub mod targets;
+
+pub mod contracts;
```

Comment:
```suggestion
extern crate alloc;
use arbitrary::{Arbitrary, Error, Result, Unstructured};

pub mod util;
pub mod targets;
pub mod contracts;

pub use targets::*;
```

I don't also see any reason to make some module public if we pub using everything from it